Basic_CNN_Architecture.py

```diff
@@ -0,0 +1,48 @@
+import torch
+import torch.nn as nn
+
+class BasicCNN(nn.Module):
+	def __init__(self, 
+			in_channels: int, 
+			enc_channels: int, 
```

Comment:
_enc_channels_ (input line 7) seems to be used both in Convolution layers(line 18,23) and in the Linear layers (31,32)

I think _enc_channels_ (line 18,23) represents number of output convolution channels for the first Convolution layer, out channels represents number of output convolutional layers after the second layer

I think we should have two separate variables for the number intermediate channels in between the convolutional layers, and the number of intermediate nodes between the two linear layers.

Reply:
Yeah I can make another parameter for that.